### PR TITLE
feat: Added parameter to ignore insignificant commits

### LINKED ISSUE #7: Version being bumped repeatedly without changes
When running `versionize` repeatedly without any commits being added in between (aside from the ones versionize creates with each release), it keeps bumping the version number and generating new commits, tags, etc...

I would like to see Verzionize recognizing this situation and refusing to bump the version in such a case.
Any thoughts about this?

I would be willing to make a PR for this if the author likes the idea.

### PR DESCRIPTION
**Pull request by AlexGoris-KasparSolutions**
This parameter allows for users to specify that Versionize should ignore insignificant commits.
Insignificant commits are any non- `feat`, `fix` or `BREAKING` commit

fixes #7

### CODE REVIEW

**Comment by AlexGoris-KasparSolutions** [comment]
I attempted to fix the AppVeyor pipeline, but failed to do so, in [this run](https://ci.appveyor.com/project/saintedlama/versionize/builds/34594802) however, it's apparent that the `$env:COVERALLS_REPO_TOKEN` variable does not exist on the system running the pipeline, which is most likely the reason it is failing.

File: README.md [hunk 1]
```diff
@@ -45,7 +45,8 @@ Options:
   -d|--dry-run                         Skip changing versions in projects, changelog generation and git commit
   --skip-dirty                         Skip git dirty check
   -r|--release-as <VERSION>            Specify the release version manually
-  --silent                             Supress output to console
```
Comment (saintedlama) [inline README.md:48]: 👍

**Comment by saintedlama** [comment]
AppVeyor problems were caused by a coveralls.net 2.0.0 beta version. Pinned the version to 1.0.0 since I could not find the beta source anywhere

**Comment by saintedlama** [comment]
Published as version 1.4 - thanks for your PR 🎉

**Pull request merged**